next.config.js

```diff
@@ -70,6 +70,10 @@ const configVals = {
                 protocol: 'https',
                 hostname: 'i.ytimg.com', // YouTube thumbnails
             },
+            {
+                protocol: 'https',
+                hostname: 'raw.githubusercontent.com',
```

Comment:
Found an article which had images that weren't loading (https://www.mongodb.com/developer/products/mongodb/mongo-socket-chat-example/) while testing these changes, so I threw this in this PR.